HHS/Head-Start-TTADP
# [TTAHUB-1729] Smart Sheet events import CSV

## Description of change

Created a script to import Smart Sheet events. This populates the ownerId, regionId, and data blob columns. If a event already exists with the event id OR a creator cannot be found the event is skipped.

**Note:** Because we don't yet have Smart Sheet API access. This is just the first step of importing a CSV file from S3.

**cmd run:**
_yarn importSmartSheetEvent -f FILE_KEY_

## How to test

1. Review the code
2. Run the unit test
3. Import the CSV to MinIO from the JIRA 1729 and make sure everything is created correctly in the DB.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1729


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
